Makefile

```diff
@@ -0,0 +1,70 @@
+REQUIRED_NODE_VERSION = $(shell cat .nvmrc)
```

Comment:
Is this guaranteed to have the necessary version? (I did not see `.nvmrc `in the commit)

Reply:
Yes, that file was pre-existing and is where the required version of node.js is tracked. Federalist also uses nvm so the official deployed build is produced with the same version.